src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/organisations/OrganisationsMappingApiService.kt

```diff
@@ -30,7 +30,7 @@ class OrganisationsMappingApiService(@Qualifier("mappingApiWebClient") webClient
     }
     .awaitFirstOrDefault(CreateMappingResult())
 
-  suspend fun createOrganisationMapping(mapping: CorporateMappingDto): CreateMappingResult<CorporateMappingDto> = createMapping("/mapping/corporate/organisation", mapping)
+  suspend fun createOrganisationMapping(mapping: CorporateMappingDto): CreateMappingResult<OrganisationMapping> = createMapping("/mapping/corporate/organisation", mapping)
```

Comment:
All mappings related to organisations always return an mapping that has a dpsId and nomisId - so this allows us to share the duplicate handling code across different DTOs that have no common class but all have the same "shape"